tests/playwright/playwright/index.scss

```diff
@@ -0,0 +1 @@
+@use '@gravity-ui/uikit/styles/styles';
```

Comment:
Can we import uikit and other styles directly from js?
`import '@gravity-ui/uikit/styles/styles.scss';`